src/constants/transformation/transform-service.ts

```
@@ -87,42 +250,108 @@ export class TransformService {
+        courseTrackingStartDate: data.createdOn ? new Date(data.createdOn) : undefined,
+        courseTrackingEndDate: data.completedOn ? new Date(data.completedOn) : undefined,
+      };
+
+      return transformedData;
+    } catch (error) {
+      console.error('Error transforming course tracker data:', error);
+      throw error;
+    }
+  }
+
+  async transformContentTrackerData(data: any, contentName: string): Promise<Partial<ContentTracker>> {
```

ESLint

[error] Replace `data:·any,·contentName:·string` with `⏎····data:·any,⏎····contentName:·string,⏎··`

(prettier/prettier)